src/App.css

```diff
@@ -13,7 +71,188 @@ nav ul {
   margin-right: auto;
 }
 
+.profile-icon {
+  border: 1px solid #e8e8e8;
+  padding: 7px 10px;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.person-icon {
+  color: #0290ff;
+}
+
 .container {
   max-width: 85%;
   margin: 0 auto;
 }
+
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  margin-bottom: 20px;
+  border: 1px solid #e8e8e8;
+  padding: 30px;
+}
+
+.title-div li {
+  display: inline-block;
+  font-size: 14px;
+  color: #0290ff;
+  font-weight: lighter;
+  cursor: pointer;
+  font-family: 'Roboto Slab', serif;
+}
+
+.title-div button {
+  background-color: white;
+  border: none;
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+  padding: 0 10px;
+  margin: 0 10px;
+  font-size: 14px;
+  color: #0290ff;
+  font-weight: lighter;
+  font-family: 'Roboto Slab', serif;
+  cursor: pointer;
+}
+
+.title {
+  margin-bottom: 8px;
+  font-family: 'Roboto Slab', serif;
+}
+
+.author {
+  font-family: 'Roboto Slab', serif;
+  margin-bottom: 15px;
+  font-size: 14px;
+  color: #0290ff;
+}
+
+.progress-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* width: 100%;
+  max-width: 200px; */
+  padding-right: 30px;
+  margin-right: 30px;
+
+  /* border: 1px solid chocolate; */
+}
+
+.progress-div {
+  /* position: absolute; */
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 4px solid #307bbe;
+  border-left-color: #e8e8e8;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+  margin-right: 15px;
+}
+
+.percentage {
+  font-family: "Montserrat", sans-serif;
+  color: #121212;
+  font-size: 2rem;
+}
```

Comment:
- [ ] Currently, the progress percentages for several books are not aligned either on the left or right. Kingly works on their alignment either on the left as marked in the screenshot or on the right so your design matches the original design given. 👍 

**Your Design**

![image](https://user-images.githubusercontent.com/50941074/208230709-164a851b-6f2a-4781-a80b-8b04a640bf7f.png)

**The Original Design**

![image](https://user-images.githubusercontent.com/50941074/208230795-0bedfced-e3a2-42a4-87f7-211b32e0e755.png)